relay-server/src/service.rs

```diff
@@ -142,10 +146,13 @@ impl ServiceState {
         let system = System::current();
         let registry = system.registry();
 
+        let main_runtime = utils::tokio_runtime_with_actix();
+        let outcome_runtime = utils::tokio_runtime_with_actix();
+        let mut _store_runtime = None;
```

Comment:
Why the leading underscore?

Reply:
It silences a warning since the runtime is unused if the `processing` feature is disabled.